### PR TITLE
Update tyk-sync.md

### PR DESCRIPTION
**Pull request by carolwkc**
## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/tyk-sync.md [hunk 1]
```diff
@@ -117,6 +117,8 @@ Flags:
       --apis               Specific api_id's selection (optional)
 ```
 
+The API secret to use with tyk-sync is your “Tyk Dashboard API Access key”.
```
Comment (letzya) [inline tyk-docs/content/tyk-sync.md]: or the gateway's?

Comment (carolwkc) [reply tyk-docs/content/tyk-sync.md]: Reworded

**Comment by letzya** [comment]
@caroltyk please resolve conflicts

**Comment by carolwkc** [comment]
@letzya resolved conflicts

**Review by letzya — APPROVED** [review]

**Pull request merged**